lib/core/src/control/event_loop.rs

```diff
@@ -50,7 +49,9 @@ impl EventLoop {
         let main = move |time_ms| { data.upgrade().map(|t| t.borrow_mut().run(time_ms)); };
         with(self.rc.borrow_mut(), |mut data| {
             data.main = Some(Closure::new(main));
-            data.run(0.0);
+            if let Some(closure) = &data.main {
+                web::request_animation_frame(&closure).unwrap();
+            }
```

Comment:
The if will be always true. What about 
```rust
let closure = Closure::new(main)
web::request_animation_frame(&closure).unwrap();
data.main = Some(closure)
```

Reply:
I updated it to
```rust
            data.main = Some(Closure::new(main));
            web::request_animation_frame(&data.main.as_ref().unwrap()).unwrap();
```

I thought about this
```rust
let closure = Closure::new(main)
web::request_animation_frame(&closure).unwrap();
data.main = Some(closure)
```

But `closure` execute these lines in `EventLoopData`'s `run`:

```rust
        let callback_id = self.main.as_ref().map_or(default(), |main| {
            callbacks.run_all(time_ms);
            web::request_animation_frame(main).unwrap()
        });
```